README.md

```diff
@@ -27,6 +27,11 @@ in this [blog post](https://blog.nlnetlabs.nl/testing-the-waters-with-krill/).
 
 # Changelog
 
+## 0.10.3 'Down Under'
+
+This release fixes an interoperability issue with the APNIC CA system which
```

Comment:
Should you say which action by users of Krill was impacted by the issue?

Reply:
I thought it would be too detailed. It's in issue #933 though.